include/irr/config/BuildConfigOptions.h.in

```diff
@@ -21,5 +21,6 @@
 
 // extra config
 #cmakedefine __IRR_FAST_MATH
+#cmakedefine _IRR_EMBED_BUILTIN_RESOURCES
```

Comment:
you're missing the magical glue that defines it for this config header (should be in `./source/Irrlicht/CMakeLists.txt`)